Helper/Adapter/FiltersHelper.php

```diff
@@ -197,11 +197,20 @@ public function getPriceFilters($storeId)
         $paramPriceSliderData = $this->getParamPriceSlider($storeId);
         $priceSlider = $paramPriceSliderData['price_slider'];
         $paramPriceSlider = $paramPriceSliderData['param'];
+        $prices = [];
 
         if (!is_null($this->request->getParam($paramPriceSlider))) {
```

Comment:
Could you maybe add comment that this condition parses prices for IS.js generated URL? (if it's correct)